gclient/src/api/calls.rs

```diff
@@ -87,7 +128,10 @@ impl GearApi {
         Err(Error::EventNotFound)
     }
 
-    /// `pallet_gear::create_program` with bytes in payload batched.
+    /// Create a batch of programs.
```

Comment:
Maybe some extra comments to explain what "batch" is

Reply:
Done